packages/chrome/src/utils/useFocusableMount.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useLayoutEffect, useRef, MutableRefObject } from 'react';
+import activeElement from 'dom-helpers/activeElement';
+
+export interface IUseFocusableMountProp {
+  isMounted?: boolean;
+  focusOnMount?: boolean;
+  restoreFocus?: boolean;
+  targetRef: MutableRefObject<HTMLElement | null>;
+}
+
+export function useFocusableMount({
+  isMounted,
+  focusOnMount,
+  restoreFocus,
+  targetRef
+}: IUseFocusableMountProp) {
+  const wasOpenRef = useRef<boolean>(false);
+  const triggerRef = useRef<HTMLElement | Element | null>();
+
+  useLayoutEffect(() => {
+    if (isMounted && focusOnMount && targetRef.current) {
+      triggerRef.current = activeElement();
+      wasOpenRef.current = true;
+      targetRef.current.focus();
```

Comment:
The `StyledSheet` is not handling focus styling properly. It should have

```css
&:focus {
  outline: none;
}
```

See `StyledDrawerModal` for details.

Reply:
The `Sheet` differs from the modality of the `DrawerModal`, do you think they should be treated the same in regards to focus? Would it be confusing for users without an indication (without screen readers)? This is a broad question to ask, however my instinct tells me to leave this to the user-agent, or be more explicit with it.

Reply:
Already covered in the RFC, the `Sheet` will offer `focusOnMount` to allow the consumer to control whether or not the `Sheet` will receive focus. In the case where a consumer determines that it should, we should not be seeing the browser's default outline.

Reply:
Fair point, will update.